doc/devel/coding_guide.rst

```diff
@@ -47,6 +47,18 @@ PR Review guidelines
 * If you have commit rights, then you are trusted to use them.  Please
   help review and merge PRs!
 
+* Tag the PR with appropriate labels.
```

Comment:
Can you link to the label list here? And maybe descriptive instead of appropriate since appropriate is subjective?